pdfrw/tokens.py

```diff
@@ -95,24 +83,23 @@ def _gettoks(self, startloc, cacheobj=_cacheobj,
         fdata = self.fdata
         current = self.current = [(startloc, startloc)]
         cache = {}
+        get_cache = cache.get
         while 1:
             for match in findtok(fdata, current[0][1]):
                 current[0] = tokspan = match.span()
                 token = match.group(1)
                 firstch = token[0]
+                toktype = intern
```

Comment:
It seems a bit weird to me that `toktype` can be the intern function, but I'll leave that up to you. I know you're more willing to trade some readability for performance than I would be. ;-)